[5.5][webapi] prepared 5.5 directory for web API reference

pkosko opened the pull request:
There was no directory for API reference for newest version (5.5),
it was created by this commit as a copy of 5.0 API reference.

This PR is an initial work for future 5.5-ACR-related work.

huytaduc commented:
checkout the webpage https://docs.stg.tizen.org/staging/532

huytaduc commented:
checkout the webpage https://docs.stg.tizen.org/staging/532

huytaduc commented:
checkout the webpage https://docs.stg.tizen.org/staging/532

mijin85cho commented on docs/application/web/api/5.5/device_api/tv/tizen/tvaudiocontrol.html:
```diff
@@ -0,0 +1,847 @@
+<p><span class="privilege">Privilege: </span>
+ http://tizen.org/privilege/tv.audio
+            </p>
+<div class="parameters">
```
It looks like #529 is missing in some files.
Is it OK not to add "deprecated since 5.0" information?

mijin85cho commented on docs/application/web/api/5.5/index.htm:
```diff
@@ -0,0 +1,846 @@
+								<li><a href="device_api/wearable/tizen/tizen.html">Tizen</a></li>
+							</ul>
+						</li>
+						<li><a href="device_api/wearable/index.html#Application">Application Framework</a>
```
Compared to "5.5/device_api/wearable/index.html" file, I can see the structure doesn't match with this list.
For example we don't have "Account", "Preference", and "Social", etc for "Wearable Web" here.
Don't we need to sync this page with actual list?

Same issue to Mobile Web and TV Web.

mijin85cho commented on docs/application/web/api/5.5/w3c_api/w3c_api_cover.html:
```diff
@@ -0,0 +1,43 @@
+
+
+<p>Click the profile you want:</p>
+<p><a href="w3c_api_m.html"><img alt="Mobile Web" src="../images/mw_division.png"/></a> <a href="w3c_api_w.html"><img alt="Wearable Web" src="../images/ww_division.png"/></a></p>
```
Is link to "w3c_api_tv.html" unnecessary?

mijin85cho commented:
On the messenger, @pkosko asked to handle these issues with other PRs, to focus on adding 5.5 API file in this PR.

As these added 5.5 files are copied version of 5.0, 
I also think it would be batter to update this PR with re-copied version of 5.0 after other PRs related with web API are merged.
Please replace the updated files with 5.0 after other PRs(#538, etc) are merged.

**Another request**
For Web API reference files before we had "index.xml" which list up the whole pages according to the Web API reference structure.
[Reference link](https://review.tizen.org/gerrit/gitweb?p=sdk/online-doc.git;a=blob;f=org.tizen.web.apireference/index.xml;h=ccaff9b6c30b16b763bd1a3f398220acfce23d17;hb=refs/heads/tizen)

We are missing it now, so please add index.xml file again from 5.5 and update it to get TOC of Web API files.

huytaduc commented:
checkout the webpage https://docs.stg.tizen.org/staging/532

pkosko commented:
1. Updated all sources from master to tizen_5.5_prepare
2. Copied 5.0 directory to 5.5 directory
3. Modified latest symbolic link
4. Added index.xml files for all versions

mijin85cho commented:
Hello, @pkosko 


It looks like this PR took every file update history on master branch.
(guides, training, tizen-studio...)

How about just focusing on the 5.5 web API files only in this PR?

huytaduc commented:
checkout the webpage https://docs.stg.tizen.org/staging/532

mijin85cho commented:
Hello, @pkosko 

Thanks for the update. 

 I have 2 queries to ask your opinion.

1. About api/5.5/index.htm file.
It looks like this file is not updated. The menu tree listed in this file doesn't match with the structure in device_api/mobile/index.html or device_api/wearable/index.html. Would you please update api/5.5/index.htm to get synced with actual menu tree?

2. The location of newly added index.xml files.
index.xml files for mobile, wearable and tv are under device_api/\<profile\> folder.
However as each files has information for other web api files also, how about locating it under "api\5.5" with renaming it index_mobile.xml, index_wearable.xml, and index_tv.xml?

Please let me know your thoughts

mijin85cho commented:
Hello, @pkosko 

We checked the links in web/api/index.md, and decided to link to device_api/mobile/index.html as the first page as same as on Tizen Developer Site. So you don't need to update api/5.5/index.htm.
Please ignore my first comment above.

mijin85cho commented:
On the messenger, we decided to keep the current location for the index.xml files.

Thanks for the updates.
